lib/livebook_web/components/form_components.ex

```diff
@@ -575,9 +575,13 @@ defmodule LivebookWeb.FormComponents do
 
   defp help(assigns) do
     ~H"""
-    <span class="cursor-pointer tooltip top" data-tooltip={@text}>
-      <.remix_icon icon="question-line" class="text-sm leading-none" />
-    </span>
+    <div class="flex items-center relative">
+      <div class="fixed">
```

Comment:
Unfortunately this is not going to work if the content exceeds the screen (a long modal). Perhaps we should change the help tooltip to be "right", making it less likely to overflow.

Reply:
Yeah, it makes more sense.